packages/wallets/src/components/WalletCard/__tests__/WalletCard.spec.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { ComponentProps } from 'react';
+import { APIProvider, useBalance } from '@deriv/api';
+import { render, screen } from '@testing-library/react';
+import WalletCard from '../WalletCard';
+
+jest.mock('@deriv/api', () => ({
+    ...jest.requireActual('@deriv/api'),
+    useBalance: jest.fn(),
+}));
+
+describe('WalletCard', () => {
+    let mockProps: ComponentProps<typeof WalletCard> = {
```

Comment:
typically its good to have explicit tests for all props and all edge cases, that also includes iconSize and landingCompanyName (which are not covered at the moment).